src/components/Speciality/Projects.js

```diff
@@ -84,7 +84,7 @@ const ProjectsSection = ({ related, title, clients }) => (
           <Col width={[1, 1, 1, 1, 1 / 2]} />
         </Row>
       ) : null}
-      {clients ? <CompaniesHelped clients={clients} /> : null}
+      {clients && clients.length ? <CompaniesHelped clients={clients} /> : null}
```

Comment:
While we refactor, in general I think it's always best to write this as:
`{clients && clients.length && <CompaniesHelped clients={clients} />}`
It removes a `: null`that we don't need.

Reply:
Actually, I have a feeling why that was like that - sometimes (not sure exactly why) if you use the `array && array.length && <Component>` approach, the logic will end up rendering literal 0 on the website in place of the component (I've had this issue).

Reply:
I changed this per your suggestion - the padding ain't pretty when one of them is missing, but I guess that's a design input that's needed as part of the conditional colours.